dev/run-python-skinny-tests.sh

```diff
@@ -33,5 +33,7 @@ pytest tests/deployments/test_cli.py
 pytest tests/deployments/test_deployments.py
 pytest tests/projects/test_projects_cli.py
 pytest tests/utils/test_requirements_utils.py::test_infer_requirements_excludes_mlflow
+pytest tests/store/tracking/test_file_store.py
```

Comment:
Add file store testing because:

1. skinny should support file store without any additional dependencies
2. test_file_store tests basic search capabilities

Reply:
great idea!